0xPolygonMiden/miden-client
# Atomic account new

Make creation of a new account atomic by using sqlite `Transactions`.

Test:
1.  Create account: 
`cargo run --release --features testing -- account new fungible-faucet -t TEST -d 10 -m 10000`
2. Verify account creation:
`cargo run --release --features testing -- account list`

Closes #20 
Closes #40
